Sources/System/FilePath/FilePath.swift

```diff
@@ -62,6 +62,18 @@ public struct FilePath {
 
 // @available(macOS 10.16, iOS 14.0, watchOS 7.0, tvOS 14.0, *)
 extension FilePath {
+  
+  /// Returns the current working directory of this process.
+  ///
+  /// - Warning: This value is global and care should be taken to make sure it does not unexpectedly change.
+  public static func getCurrentWorkingDirectoryForProcess() throws -> FilePath {
```

Comment:
We should lose the `get` prefix -- per the Swift API design guidelines, functions without side effects should read as noun phrases. 

I know you specifically added this in response to discussions, but I would like to also remove the `ForProcess` suffix. There are two reasons for this: 

- First, I don't think it makes sense to try documenting the precise semantics of these functions through their naming -- it seems futile to me, and it leads to an unwieldy naming scheme.

- Far, far more importantly, while the default semantics of these syscalls have been pretty stable, there is no reason an OS cannot switch to different behavior in certain circumstances. For instance, on Linux and macOS at least [we have non-portable ways](https://stackoverflow.com/a/55428238/77918) to set up a thread-local working directory.

```suggestion
  public static func currentWorkingDirectory() throws -> FilePath {
```